app.js

```diff
@@ -239,7 +283,7 @@ function processArgs(args, options) {
       configFile: {
         description: 'Path to a SAML attribute config file',
         required: true,
-        default: require.resolve('./config.js'),
+        default: 'saml-idp/config.js',
```

Comment:
same comment as above, we shouldn't hard-code `saml-idp` path and use a process variable instead